arch/arm/soc/st_stm32/stm32l0/Kconfig.series

```diff
@@ -0,0 +1,21 @@
+# Kconfig - ST Microelectronics STM32L4 MCU series
+#
+# Copyright (c) 2016 Open-RnD Sp. z o.o.
+# Copyright (c) 2016 BayLibre, SAS
+# Copyright (c) 2018 Endre Karlson <endre.karlson@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_SERIES_STM32L0X
+	bool "STM32L0x Series MCU"
+	select CPU_CORTEX_M
+	select CPU_CORTEX_M0PLUS
+	select SOC_FAMILY_STM32
+	select SYS_POWER_LOW_POWER_STATE_SUPPORTED
+	select HAS_STM32CUBE
+	select CPU_HAS_MPU
```

Comment:
Not all L0s have MPU. Let's do that on per-SOC basis. L073 has it.

Reply:
Ok, we can do that in seperate PRs then..